dual_net.py

```diff
@@ -274,10 +276,13 @@ def train_ops(input_tensors, output_tensors, **hparams):
             labels=input_tensors['pi_tensor']))
     value_cost = tf.reduce_mean(tf.square(
         output_tensors['value_output'] - input_tensors['value_tensor']))
-    l2_cost = 1e-4 * tf.add_n([tf.nn.l2_loss(v)
-                               for v in tf.trainable_variables() if not 'bias' in v.name])
+    l2_cost = hparams['l2_strength'] * tf.add_n([tf.nn.l2_loss(v)
+                                                 for v in tf.trainable_variables() if not 'bias' in v.name])
     combined_cost = policy_cost + value_cost + l2_cost
-    learning_rate = tf.train.exponential_decay(1e-2, global_step, 10 ** 7, 0.1)
+    boundaries = [1e6, 2 * 1e6]
```

Comment:
Some more magic variables that would be nice to comment (maybe these should be in hyperparams).